README.md

```diff
@@ -101,74 +31,98 @@ unless real values don't help users know what to change.
 -->
 
 ```hcl
-
+provider "ibm" {
+  ibmcloud_api_key = ""
+  region           = "us-south"
+}
+module "icd_elasticsearch" {
+  source  = "terraform-ibm-modules/icd-elasticsearch/ibm"
+  region                   = var.region
+  version = "latest" # Replace "latest" with a release version to lock into a specific release
+  resource_group_id    = "xxXXxxXXxXxXXXXxxXxxxXXXXxXXXXX"
+  region               = "us-south"
+}
 ```
 
 ## Required IAM access policies
```

Comment:
This needs to be a H3 heading, as per the new template. Please update `### Required IAM access policies`